pyathena/sqlalchemy_athena.py

```diff
@@ -31,6 +31,34 @@
 
 import pyathena
 
+BLANKS = re.compile(r"\s+")
+LIMIT_COMMENT_COLUMN = 255
+# https://docs.aws.amazon.com/athena/latest/APIReference/API_Column.html
+LIMIT_COMMENT_TABLE = 51200
+# https://docs.aws.amazon.com/athena/latest/APIReference/API_TableMetadata.html
+
+
+def process_comment_literal(value, dialect, squash_blanks=False):
+    """Escapes comments in DDL statements
+
+    String literals in COMMENT are not escaped in the same way as data
+    literal strings. Data literal use '' (double single quotes), comments escape
+    single quotes with a \\ (backslash).
+    Additionally, column comments do not support the presence of new line, carriage
+    return, etc. characters while table comments do. Hence the ``squash_blanks``
+    argument.
+    Finally, % (percent signs) need to be escaped, by being doubled, so as to not
+    be confused with placeholders for query parameters.
+    """
```

Comment:
Change the name of the method so that you can understand what is being processed. No comments are required.